### PR TITLE
New sample: dynamic commands for faster development.

### PR DESCRIPTION
**Pull request by davidharcombe**
How to create dynamically loaded commands from Cloud Storage to assist in faster development and prototyping.

### CODE REVIEW

File: python/dynamic-commands/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Google Chat App Dynamic Commands
+
+This sample shows how to load dynamic commands for a Python Cloud Function-based
+Chat App from a Google Cloud Storage Bucket to allow for a faster prototyping
```
[the remark this answers is not in the document]
Comment (davidharcombe) [reply python/dynamic-commands/README.md:4]: There's no need to redeploy so it saves up to 3 minutes per change.

File: python/dynamic-commands/install.sh [hunk 2]
```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (davidharcombe) [reply python/dynamic-commands/install.sh:1]: Maybe do both?

File: python/dynamic-commands/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpoehnelt) [reply python/dynamic-commands/README.md:4]: If speed is the reason, why not suggest a proxy/tunnel approach? https://github.com/anderspitman/awesome-tunneling

[the remark this answers is not in the document]
Comment (davidharcombe) [reply python/dynamic-commands/README.md:4]: I'm also not sure this is possible with Cloud Functions

**Review by sqrrrl — APPROVED** [review]

**Pull request merged**